ADALiOS/ADALiOS/ADBrokerKeyHelper.m

```diff
@@ -23,7 +23,7 @@
 #import "ADKeyChainHelper.h"
 #import <CommonCrypto/CommonCryptor.h>
 #import <Security/Security.h>
-#import "ADLogger.h"
+#import "ADLogger+Internal.h"
```

Comment:
Yup, that's the naming convention for categories. (Which are a way in Objective-C of extending a class, or splitting up a class's implementation across multiple files.)